lib/diagnostic.rb

```diff
@@ -89,7 +126,11 @@ def example
     Example.first
   end
 
-at  describe 'DELETE destroy' do
-    # your test(s) here
+  describe 'DELETE destroy' do
+    it 'is successful and returns an empty response' do
```

Comment:
Just make sure you're passing the `example`'s `id` to the `delete :destroy` action.

`delete :destroy, example: example.id`